vnext/JSI/Desktop/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="boost" version="1.72.0.0" targetFramework="native" />
```

Comment:
>boost [](http://example.com/codeflow?start=15&length=5)

I'm curious what this is needed for?

Reply:
Indirect dependencies of the JSI Runtime holder. (Folly, I believe).